tests/_support/Helper/Acceptance.php

```diff
@@ -18,4 +18,19 @@ public function sendEmail($to, $subject, $body)
         $phpmailer->Body = $body;
         $phpmailer->send();
     }
+
+    public function sendEmailWithEncoding($to, $subject, $body, $encoding)
```

Comment:
Lots of duplication here, is it worth just updating the above method `sendEmail` and setting a default for the encoding?